src/components/app/app.ts

```diff
@@ -27,8 +27,8 @@ export class App {
     public async initialize(): Promise<void> {
         const settings = await this.settingsProvider.getSettings();
 
-        if (!settings["managementApiUrl"]) {
-            this.viewManager.addToast(startupError, `Management API URL is missing. See setting <i>managementApiUrl</i> in the configuration file <i>config.design.json</i>`);
+        if (!settings[SettingNames.backendUrl]) {
+            this.viewManager.addToast("Warning", WarningBackendUrlMissing);
```

Comment:
The behaviour of the startup is not changed. Here toast will be shown to user that backend url is not set so it will be user-friendly notification about why self-hosted portal is not working or not retrieving any data.